src/Footer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+
+class Footer extends React.Component {
+
+  render() {
+    const auth = this.props.auth;
+
+    const links = [
```

Comment:
This whole file seems slightly confused. Probably need some style/best practices feedback on this.